pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -94,7 +94,11 @@ cpdef value_count_{{dtype}}({{c_type}}[:] values, bint dropna):
     build_count_table_{{dtype}}(values, table, dropna)
     {{endif}}
 
+    {{if dtype == 'object'}}
```

Comment:
Need the if condition here, for dtypes like `uint` which don't have a Python builtin equivalent.

Reply:
Alternative might be to do `np.dtype('{{dtype}}')` (but didn't check the exact names whether that would work for all)

Reply:
That compiles at least. Trying it out.